wandb/sdk/launch/utils.py

```diff
@@ -181,6 +182,8 @@ def construct_launch_spec(
         launch_config,
     )
     launch_spec["entity"] = entity
+    if author:
```

Comment:
we want the ability to overload the author here, so that we can pass through the original author from a sweep scheduler, which does NOT necessarily equal the author that runs the LaunchAgent.